Fix randomness for unit logic vote test

### Expected behavior

Test always pass.

### Actual behavior

Test has failed once in line https://github.com/LiskHQ/lisk/blob/1.2.0/test/unit/logic/vote.js?utf8=%E2%9C%93#L416.
```
10:17:31 >> AssertionError: expected 'Failed to add vote, delegate "genesis_99" already voted for' to equal 'Failed to add vote, delegate "genesis_95" already voted for'
10:17:31 >>   at Immediate.vote.checkConfirmedDelegates.err (/home/lisk/workspace/lisk-core_PR-2203-ACJRJ4PWFBHQVWOMLPKHG5V6I6EL3BSFCRNIQFR5DGA2XZH6JJGQ/test/unit/logic/vote.js:416:20)
10:17:31 >>   at runCallback (timers.js:674:20)
10:17:31 >>   at tryOnImmediate (timers.js:645:5)
10:17:31 >>   at processImmediate [as _immediateCallback] (timers.js:617:5)
10:17:31 Test failed: test/unit/logic/vote.js
```

### Steps to reproduce

https://jenkins.lisk.io/job/lisk-core/job/PR-2203/7/

### Which version(s) does this affect? (Environment, OS, etc...)

`1.2.0`

--- a/logic/account.js
+++ b/logic/account.js
@@ -327,6 +327,7 @@ class Account {
 				'missedBlocks',
 				'vote',
 				'publicKey',
+				'address',
 			];
 			sort = sortBy.sortBy(filter.sort, {
 				sortFields: allowedSortFields,

--- a/modules/delegates.js
+++ b/modules/delegates.js
@@ -419,6 +419,7 @@ __private.checkDelegates = function(senderPublicKey, votes, state, cb, tx) {
 					{
 						address: voteAddressesWithActions.map(({ address }) => address),
 						isDelegate: 1,
+						sort: 'address:desc',
 					},
 					(err, votesAccounts) => {
 						if (err) {

--- a/test/unit/logic/account.js
+++ b/test/unit/logic/account.js
@@ -510,6 +510,22 @@ describe('account', () => {
 					done();
 				});
 			});
+
+			it('should sort the result according to address in ascending order', done => {
+				account.getAll({ sort: 'address:asc' }, ['address'], (err, res) => {
+					expect(err).to.not.exist;
+					expect(res).to.eql(_.sortBy(res, 'address'));
+					done();
+				});
+			});
+
+			it('should sort the result according to address in descending order', done => {
+				account.getAll({ sort: 'address:desc' }, ['address'], (err, res) => {
+					expect(err).to.not.exist;
+					expect(res).to.eql(_.sortBy(res, 'address').reverse());
+					done();
+				});
+			});
 		});
 
 		describe('sort using object as argument', () => {

